frontend/src/components/icons/IconBase.vue

```diff
@@ -8,7 +8,7 @@ SPDX-License-Identifier: Apache-2.0
   <svg xmlns="http://www.w3.org/2000/svg"
     :width="width"
     :height="height"
-    :view-box="viewBox"
+    :viewBox="viewBox"
```

Comment:
I think we should add a comment that this must be camelCase.

Reply:
agreed & done